docs/README.md

```diff
@@ -1,91 +1,102 @@
 
-Prop types are: 
+```json
+{
+  "store.home": {
+    "children": ["newsletter-form"]
+  },
 
-- `string` 
-- `enum` 
-- `number` 
-- `boolean` 
-- `object` 
-- `array` 
+  "newsletter-form": {
+    "children": [
+      "newsletter-input-email",
+      "newsletter-input-name",
+      "newsletter-submit"
+    ]
+  }
+}
+```
 
-When documenting a prop whose type is `object` or `array` another prop table will be needed. You can create it following the example below:
+### `newsletter-form` props
 
-- `propName` object:
+| Prop name      | Type    | Description                                                                                                                                                               | Default value |
+| -------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| `ErrorState`   | `block` | Block to be rendered if the submission of the form fails. If none is passed, a default error component will be rendered.                                                  | `undefined`   |
+| `SuccessState` | `block` | Block to be rendered if form submission succeeds. If none is passed, a default success component will be rendered.                                                        | `undefined`   |
+| `LoadingState` | `block` | Block to be rendered while the form submission is loading. If none is passed, the default behavior is for the submit button to show a spinner during this loading period. | `undefined`   |
 
-| Prop name    | Type            | Description    | Default value                                                                                                                               |
-| ------------ | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ---------- | 
-| `XXXXX`      | `XXXXXX`       | XXXXXXXX         | `XXXXXX`        |
+### `newsletter-input-email` props
 
+| Prop name         | Type     | Description                                    | Default value                                                              |
+| ----------------- | -------- | ---------------------------------------------- | -------------------------------------------------------------------------- |
+| `placeholderText` | `string` | Placeholder text for email input.              | `"Enter your email address"` (translated according to the store's locale). |
+| `inputLabel`      | `string` | Label for the email input.                     | `""`                                                                       |
+| `errorMessage`    | `string` | Error message to be shown if email is invalid. | `Invalid email address"` (translated according to the store's locale).     |
```

Comment:
```suggestion
| Prop name         | Type     | Description                                    | Default value                                                              |
| ----------------- | -------- | ---------------------------------------------- | -------------------------------------------------------------------------- |
| `placeholderText` | `string` | Placeholder text for email input.              | `Enter your email address` (translated according to the store's locale). |
| `inputLabel`      | `string` | Label for the email input.                     | `""`                                                                       |
| `errorMessage`    | `string` | Error message to be shown if email is invalid. | `Invalid email address` (translated according to the store's locale).     |
```

Reply:
The `""` default value means that no label is displayed?

Reply:
Yep! It's supposed to mean "empty string". Should I change it to `undefined`? 🤔